### DISCUSSION TITLE
Open Global Glacier Model (OGGM) Educational Website

### DISCUSSION THREAD

**Discussion by kmpaul** [ProjectPythia/projectpythia.github.io#62] [Ideas]
Here is a great example of an education-focused website specifically for the Open Global Glacier Model (OGGM) users.

https://edu.oggm.org/en/latest/

And the source repository is here:

https://github.com/OGGM/oggm-edu

I point this to everyone because it is an excellent example of all of the different kinds of *content* that could be on the Pythia Portal.  Check it out!  What do people think?

**Comment by clyne** [comment]
Thanks for sharing this @kmpaul. I think the OGCM group did a really nice job of organizing their material and its worth taking a look at for inspiration. Things I liked:

1. Well organized, very clean, and easy to navigate
2. A mixture of content formats (notebooks, images, and interactive apps)
3. Really good guidance on what the site is for, who the target audience is, and how to use the site. See [here](https://edu.oggm.org/en/latest/why_oggmedu.html), in particular
4. Clear and detailed instructions on how to engage on the main page (See "Get in touch")

One other thought:

What is the roll of binder-ized (vs pre-rendered) notebooks? I'm not sure the time it takes to launch and then execute a notebook buys much unless the reader is given some guidance on how to interact (make changes to the content). A science lesson might suggest playing with the value of some physical parameter and observing the effects. That would be really powerful. The handful of notebooks that I looked at didn't seem to offer any such suggestions, which made me think the potential of interactivity was being wasted. I have not had a lot of experience with binderized notebooks, so maybe I'm simply overlooking something. But it seems that if one is going to go through the trouble of producing one it's worth suggesting which knobs to turn.
